README

```diff
@@ -1,8 +1,12 @@
+[![Build Status](https://drone.io/github.com/thaohien1812/ibus-unikey/status.png)](https://drone.io/github.com/thaohien1812/ibus-unikey/latest)
+
```

Comment:
Toàn bộ file này không thể merge được vì thay đổi các thông tin cơ bản của dự án.